### PR TITLE
Add package information to __init__.py

### PR DESCRIPTION
**Pull request by PetervB**
As discussed with Johan Pouwelse we added package information to the __init__.py files.

### CODE REVIEW

**Comment by synctext** [comment]
add to whitelist

**Comment by synctext** [comment]
retest this please

File: Tribler/community/demers/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+'''
+The demers package contains TTTODO: what does this do?
```
Comment (NielsZeilemaker) [inline Tribler/community/demers/__init__.py]: The demers package contains a Dispersy community which can be used to verify the speed of message dissemination

File: Tribler/community/metadata/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+'''
+The metadata package contains TTTODO: what does this do?
```
Comment (NielsZeilemaker) [inline Tribler/community/metadata/__init__.py]: The metadata package contains a Dispersy community which can be used to disseminate metadata of torrents

File: Tribler/community/search/__init__.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+'''
+The search package contains TTTODO: what does this do?
```
Comment (NielsZeilemaker) [inline Tribler/community/search/__init__.py]: The search package contains a Dispersy community which is used to implement decentralized search in Tribler. It also allows peers to discover new .torrents.

File: Tribler/community/channel/__init__.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+'''
+The channel package contains TTTODO: what does this do?
```
Comment (NielsZeilemaker) [inline Tribler/community/channel/__init__.py]: The channel package contains a Dispersy community which is used to implement Channels. Channels are lists "favorite" .torrents create by one or more users.

File: Tribler/community/allchannel/__init__.py [hunk 5]
```diff
@@ -0,0 +1,4 @@
+'''
+The allchannel package contains contains code for a single community 
+that all Tribler members join and use to disseminate .torrent files.
```
Comment (NielsZeilemaker) [inline Tribler/community/allchannel/__init__.py]: Not true, the AllChannel community is used to collect votes for channels, and thereby discover which channels are most popular.

**Comment by PetervB** [comment]
Included comments of @NielsZeilemaker into the __init__.py files.

File: Tribler/Main/Utility/Feeds/__init__.py [hunk 6]
```diff
@@ -1,2 +1,6 @@
 # Written by Niels Zeilemaker
 # see LICENSE.txt for license information
+
+'''
+De Feeds package contains different feeds, rss and directory.
```
Comment (NielsZeilemaker) [inline Tribler/Main/Utility/Feeds/__init__.py]: typo

File: Tribler/Core/__init__.py [hunk 7]
```diff
@@ -1,5 +1,10 @@
 # Written by Arno Bakker
 # see LICENSE.txt for license information
+
+'''
+The Core package contains the core functionalities of the tribler project
```
Comment (NielsZeilemaker) [inline Tribler/Core/__init__.py]: Tribler

**Comment by NielsZeilemaker** [comment]
@PetervB next time if change multiple files, you can add them to a single commit. Only create separate commits if you're really doing something different.

**Comment by PetervB** [comment]
@NielsZeilemaker I changed these using the git gui to make it easy to edit the files (though this makes for many commits), I'll make sure to bundle them next time :)

**Pull request merged**